### PR TITLE
[WIP] added last year sponsers

### PR DESCRIPTION
**Pull request by Anupam-dagar**
live link
http://anupamdagar.me/hack-in-the-north-preview-repo/

will this be fine, I need to adjust alignment but my major concert is the logos with white background (twilio, geeksforgeeks, wolframaplha) also gitlab logo's text is quite not visible.
Review required.

### CODE REVIEW

**Comment by gauthamzz** [comment]
Doesn't look aligned. Everything else seems fine.

Get rid of the purple. use same colors. And logos in white bg make them transparent. A google search might help you. 

or

make everything in a white background.  Your call :smile:

File: static/css/style.css [hunk 1]
```diff
@@ -1,5 +1,7 @@
 /* HackRU Fall 2017 Custom Stylesheet */
```
Comment (gauthamzz) [inline static/css/style.css]: remove this damn comment. and all others of this type. How silly of us.

File: index.html [hunk 2]
```diff
@@ -105,7 +105,90 @@
 			</div>
 		</div>
 		<!-- END OF HERO SECTION -->
-
+		<div class="row min-full-height theme-purple">
+        <div class="col-10 offset-1">
+            <div class="row row-title-box">
+                <div class="col-12">
+                    <h2 class="row-title" id="about-us">Earlier Sponsers</h2>
```
Comment (gauthamzz) [inline index.html]: Past Sponsors is a better term i suppose

File: index.html [hunk 3]
```diff
@@ -105,7 +105,90 @@
 			</div>
 		</div>
 		<!-- END OF HERO SECTION -->
-
+		<div class="row min-full-height theme-purple">
```
Comment (gauthamzz) [inline index.html]: Do lint the code as well

File: index.html [hunk 4]
```diff
@@ -105,7 +105,90 @@
...
+            		<img src="2017/static/images/sponsors/twilio.png" class="img-fluid">
+            	</div>
+            	<div class="col-3">
+            		<img src="2017/static/images/sponsors/temboo.png" class="img-fluid">
+            	</div>
+            	<div class="col-3">
+            		<img src="2017/static/images/sponsors/namecheap.png" class="img-fluid">
+            	</div>
+            	<div class="col-3">
+            		<img src="2017/static/images/sponsors/geeksforgeeks.png" class="img-fluid">
```
Comment (gauthamzz) [inline index.html]: remove geeksforgeeks btw

Comment (Anupam-dagar) [reply index.html]: noted

File: static/css/style.css (hunk 1 above)
Comment (Anupam-dagar) [reply static/css/style.css]: ohh, yeah this is much needed.

**Comment by Anupam-dagar** [comment]
I will have to look for making the background white, it's the same thing which we were facing for footer section.I will do the required changes.

**Pull request merged**